content/en/docs/Installation/packages.md

```diff
@@ -28,10 +28,11 @@ More packages available, with links to download/install instructions:
 |-------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Cloudron                | https://www.cloudron.io/store/org.navidrome.cloudronapp.html                                                                                                                  |
 | Fedora                  | https://copr.fedorainfracloud.org/coprs/lchh/navidrome/                                                                                                                       |    
-| Kubernets Chart         | https://artifacthub.io/packages/helm/emmas-charts/navidrome                                                                                                                   | 
+| Helm Chart              | https://artifacthub.io/packages/helm/emmas-charts/navidrome                                                                                                                   |
+| TrueCharts Helm Chart   | https://truecharts.org/charts/stable/navidrome/                                                                                                                               | 
 | OpenMediaVault          | [Instructions using docker-compose](https://forum.openmediavault.org/index.php?thread/36635-how-to-install-navidrome-using-docker-compose-an-airsonic-booksonic-alternative/) |
 | QNAP                    | https://www.myqnap.org/product/navidrome/                                                                                                                                     |
-| TrueNAS SCALE           | https://truecharts.org/charts/stable/navidrome/                                                                                                                               |
+| TrueNAS SCALE           | https://www.truenas.com/docs/truenasapps/communityapps/navidrome/                                                                                                             |
```

Comment:
This seems to be just a placeholder, there's no information on how to install Navidrome. Don't think it is worth to keep it.

Reply:
They don't have much info on the Website for installation but I think it's a massive plus for you guys to list it. It's featured among their catalog apps and is well maintained by @stavros-k 

![image](https://github.com/user-attachments/assets/7d8b7279-4419-43b7-83f0-d6fd4276fa45)

And their installer is pretty straightforward

![image](https://github.com/user-attachments/assets/5ee1163b-a2f1-45a2-bccb-c29291b3aa83)